pkg/streams/streams.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package streams
+
+import (
+	"strings"
+
+	galasaErrors "github.com/galasa-dev/cli/pkg/errors"
+)
+
+func validateStreamName(streamName string) (string, error) {
+
+	var err error
+	streamName = strings.TrimSpace(streamName)
+
+	if streamName == "" {
+		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_MISSING_STREAM_NAME_FLAG)
+	}
```

Comment:
Missing better validation here? We should check that the stream is limited to alphanumeric characters and '_', '-' like the story mentions?